0014-runtime-signing-tx-with-hardware-wallet.md

```diff
@@ -682,57 +695,70 @@ In this case the hardware wallet renders the following UI.
 ```
 <!-- markdownlint-enable line-length -->
 
-### Signing encrypted transactions
+### Signing EVM runtime transactions
 
-[Encrypted transactions][runtime-sdk envelope] (`tx.call.format != 0`) contain
-the call data encrypted with an ephemeral X25519DeoxysII key. The hardware
-wallet is not expected to implement this scheme and decrypt the transaction,
-neither it is safe to share the ephemeral key with anyone. Instead, the user
-must enable **blind signing** and the hardware wallet should show the hash of
-the encrypted transaction, the X25519DeoxysII public key and the nonce:
+#### Creating smart contract
+
+[`evm.Create`] method will not be managed by the hardware wallet because the
+size of the EVM byte code may easily exceed the wallet's encrypted memory size.
+
+<!-- markdownlint-disable-next-line no-duplicate-header -->
+#### Calling smart contract
+
+In contrast to `contracts.Call`, [`evm.Call`] method requires contract ABI and
+support for RLP decoding in order to extract argument names from
+`tx.call.body.data`. This is outside of the scope of this ADR and the **blind
+signing**, explicitly allowed by the user, is performed.
+
+We propose the following UI:
 
 <!-- markdownlint-disable line-length -->
 ```ledger
-| Review Encrypted > | < BLIND > | < Tx hash (1/1) > | < Public key (1/1) > | <  Nonce (1/1) > | <    Fee    > | < Gas limit > | <  Network  > | <  ParaTime > | <             > | <             |
-|   Transaction      |  SIGNING! |     <TX_HASH>     |     <PUBLIC_KEY>     |      <NONCE>     |  <SYM> <FEE>  |  <GAS LIMIT>  |   <NETWORK>   |   <RUNTIME>   |     APPROVE     |     REJECT    |
-|   (ParaTime)       |           |                   |                      |                  |               |               |               |               |                 |               |
+| Review Contract > | < BLIND > | < Tx hash (1/1) > | < Address (1/1) > | <   Amount    > | <     Fee     > | < Gas limit > | <  Network  > | <  ParaTime > | <            > | <             |
+|      Call         |  SIGNING! |     <TX_HASH>     |     <ADDRESS>     | <SYM> <AMOUNT>  |   <SYM> <FEE>   |  <GAS LIMIT>  |   <NETWORK>   |   <RUNTIME>   |    APPROVE     |     REJECT    |
+|   (ParaTime)      |           |                   |                   |                 |                 |               |               |               |                |               |
 ```
 <!-- markdownlint-enable line-length -->
 
-`TX_HASH` is a hex representation of sha256 checksum of `tx.call.data` field.
+`TX_HASH` is a hex representation of sha256 checksum of `tx.call.body.data`
+field.
 
-`PUBLIC_KEY` is a hex representation of the 32-byte `tx.call.pk` field.
+`ADDRESS` is a hex-encoded address of the smart contract.
 
-`NONCE` is a hex representation of the 15-byte `tx.call.nonce` field.
+### Signing encrypted runtime transactions
 
-### Signing EVM transactions
+Encrypted transactions (`tx.call.format != 0`) contain the call data inside the
```

Comment:
This should not be `tx.call.format != 0` but rather `tx.call.format == 1` as you can't be sure that all future formats will be for encryption.